src/app/modules/candidate-registration/services/candidate-registration.service.ts

```diff
@@ -10,7 +10,7 @@ export class CandidateRegistrationService {
   /**
    * Base url to hit the backend api
    */
-  private baseUrl = 'https://0nn472c959.execute-api.ap-south-1.amazonaws.com/dev';
+  private baseUrl = 'https://jzoznlcnii.execute-api.ap-south-1.amazonaws.com/dev/registercandidatedetails';
```

Comment:
`/registercandidatedetails `, this should not be the part of BaseURL, since we will be keeping the candidate list etc services also under this file.

Assuming `/registercandidatedetail` this will get changed by something else

Reply:
ok, will add this in method